pkg/machines/reducers.js

```diff
@@ -105,8 +105,8 @@ function networks(state, action) {
             return [...state, network];
         }
 
-        const updatedNetwork = Object.assign({}, state[index], network);
-        return replaceResource({ state, updatedNetwork, index });
```

Comment:
This change here doesn't offer much to be honest. We use the updateVm constant name as well for example.

Reply:
oh now I noticed. I don't mind, we 'll probably need to merge all the functions in the future anyway, since it's code duplication.

Reply:
In the end i changed to the second way, the way updatedVm is done, to keep consistency.